tests/scripts/thread-cert/node.py

```diff
@@ -171,12 +171,10 @@ def stop_otbr_service(self):
         self.bash('service otbr-agent stop')
 
     def stop_mdns_service(self):
-        self.bash('service avahi-daemon stop')
-        self.bash('service mdns stop')
+        self.bash('(service avahi-daemon stop; service mdns stop; true')
```

Comment:
Seems there's an extra '(' in the command.

Reply:
BTW why `service avahi-daemon stop || service mdns stop` doesn't work?

Reply:
If both avahi-daemon and mdns are present and started, this ensures both are stopped.